packages/core/src/baseclient.ts

```diff
@@ -522,7 +522,7 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
           throw new SentryError('`beforeSend` returned `null`, will not send event.');
         }
 
-        const session = scope && scope.getSession();
+        const session = scope && scope.getSession && scope.getSession();
```

Comment:
What about simply using `?.`?
```js
const session = scope && scope.getSession?.();
```

Is its compatibility enough for you?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Browser_compatibility

Reply:
Yes, we could do that, as we're compiling it down to ES5 anyway. Good catch.

Reply:
For that matter, couldn't we just do

`const session = scope?.getSession?.();`

?